### ISSUE TITLE
Cannot parse Deceased field

### ISSUE THREAD

**Issue by rgreinho** [scrapd/scrapd#51]
Issue Type
----------

- Bug report

Current Behavior
----------------

The following errors are logged by ScrAPD:
```
Cannot parse Deceased: ['Luis', 'Angel', 'Tinoco,', 'Hispanic', 'male', '(11-12-07']
Cannot parse Deceased: ['Brianna', 'Nicole', 'Polzine,', 'White', 'female', '(10-28-93']
Cannot parse Deceased: ['Eva', 'Marie', 'Gonzales,', 'W/F,']
Cannot parse Deceased: ['Unidentified', 'female', 'Unidentified', 'male']
Cannot parse Deceased: ['Christopher', 'M', 'Peterson', 'W/M', '10-8-1981']
Cannot parse Deceased: ['Christopher', 'M', 'Peterson', 'W/M', '10-8-1981']
Cannot parse Deceased: ['Starlite', 'Stewart,', 'White', 'male,', '29', 'years', 'of', 'age']
Cannot parse Deceased: ['David', 'Adam', 'Castro,', 'Hispanic', 'male,', '28', 'years', 'of', 'age']
Cannot parse Deceased: ['Latulippe,', 'Nancy,', 'White', 'female,', '38', 'years', 'of', 'age', 'Latulippe,', 'Jackson,', 'White', 'male,', '14', 'years', 'of', 'age', 'Latulippe,', 'Scott,', 'White', 'male,', '41', 'years', 'of', 'age']
Cannot parse Deceased: ['Delta', 'Olin,', 'White', 'female,', '49', 'years', 'of', 'age']
Cannot parse Deceased: ['Unidentified', 'White', 'male']
Cannot parse Deceased: ['David', 'Adam', 'Castro,', 'Hispanic', 'male,', '28', 'years', 'of', 'age']
Cannot parse Deceased: ['Ronnie', 'Lee', 'Hall,', 'White', 'male,', '8-28-51']
```

Expected Behavior
-----------------

Find a way to parse them all.

Possible Solution
-----------------

Not sure there is one... 🤔

**Comment by rgreinho** [comment]
`Cannot parse Deceased: ['Christopher', 'M', 'Peterson', 'W/M', '10-8-1981']` -> https://austintexas.gov/news/traffic-fatality-65-1

**Comment by rgreinho** [comment]
Closed by #90.
